src/components/ListLayout.jsx

```diff
@@ -61,6 +61,94 @@ const ListLayout = ({ items, localToken }) => {
     return timeCheck;
   }
 
+  // updates isActive property of item to true if item has 2+ purchases and has been purchased within calculated estimate
+  // isActive is defaulted to false when item is added
+  items.forEach((item) => {
```

Comment:
Thank you!! We really didn't want this to run on page load but I was hesitant with which dependency to use because with items the function updates the isActive property in items I thought it might create an infinite loop but I just went for implementing it, React also suggested localtoken and currentTime as dependencies and I think it works really well now!